CMake/fletch-tarballs.cmake

```diff
@@ -302,10 +302,10 @@ endif()
 list(APPEND fletch_external_sources Qt)
 
 # OpenCV
-# Support 2.4.13 and 3.4 optionally
+# Support 2.4.13, 3.4, and 4.2 optionally
```

Comment:
Maybe not the time to tackle this issue but do we really need to keep 2.4.13 around?

Reply:
That is an excellent question which I do not consider myself qualified to answer :slightly_smiling_face:. Maybe a question for @mattdawkins and @mleotta?